### ISSUE TITLE
Crash 

### ISSUE THREAD

**Issue by gitkald** [stefan-niedermann/nextcloud-deck#477]
I can start the app and it worked well. But I if send the app in background, work with other apps, than the app crash.

OS: Android 6, Lenovo Tab

```
OS Version: 3.10.72+(A10-70F_S000216_170601_ROW)
OS API Level: 23
Device: A10-70F
Manufacturer: Lenovo
Model (and Product): Lenovo TAB 2 A10-70F (LenovoTAB2A10-70F)

---

java.lang.RuntimeException: Unable to start activity ComponentInfo{it.niedermann.nextcloud.deck.play/it.niedermann.nextcloud.deck.ui.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Long it.niedermann.nextcloud.deck.model.Board.getLocalId()' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2583)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2665)
	at android.app.ActivityThread.-wrap11(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1499)
	at android.os.Handler.dispatchMessage(Handler.java:111)
	at android.os.Looper.loop(Looper.java:207)
	at android.app.ActivityThread.main(ActivityThread.java:5765)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:679)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Long it.niedermann.nextcloud.deck.model.Board.getLocalId()' on a null object reference
	at it.niedermann.nextcloud.deck.ui.MainViewModel.getCurrentBoardLocalId(MainViewModel.java:27)
	at it.niedermann.nextcloud.deck.ui.stack.StackFragment.onCreateView(StackFragment.java:69)
	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2698)
	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:320)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1187)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1356)
	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1434)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1497)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2625)
	at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:2577)
	at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:247)
	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:541)
	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:201)
	at it.niedermann.nextcloud.deck.ui.branding.BrandedActivity.onStart(BrandedActivity.java:53)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1245)
	at android.app.Activity.performStart(Activity.java:6325)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2540)
	... 9 more

```

**Comment by stefan-niedermann** [comment]
Why did you cut off the error message though? Those are information we need to hunt down the bugs. Which version exactly are you using of the Nextcloud Android app, Nextcloud Deck Android app and of the Nextcloud Deck server app? Which flavor are you using (F-Droid or Play Store)?

Can you try to clear the storage and then readd your account in the first place? (Android settings → Apps → Nextcloud Deck → Storage → Delete storage)

**Comment by stefan-niedermann** [comment]
Never mind, i found the root cause of the issue and we will track and fix this bug in #478 . Next version will solve this bug.

Thank you for the report!

But for the next time reporting an issue please keep the error message intact :v:

**Comment by gitkald** [comment]
Hi Stefan, thanks 4 your answer. I use your App Button to copy the log information. I didn't see moren lines, like those I posted.

And it was a fresh install from google playstore. 

Same issue after deleting the storrage.
